cmd/keycmd/print.go

```diff
@@ -0,0 +1,53 @@
+package keycmd
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/key"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	avago_constants "github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/olekukonko/tablewriter"
+)
+
+func printAddresses(keyPaths []string) error {
+	header := []string{"Key Name", "Chain", "Address", "Network"}
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader(header)
+	table.SetRowLine(true)
+	table.SetAutoMergeCells(true)
+
+	supportedNetworks := map[string]uint32{
+		models.Local.String(): 0,
+		models.Fuji.String():  avago_constants.FujiID,
+		/*
+			Not enabled yet
+			models.Mainnet.String(): avago_constants.MainnetID,
+		*/
+	}
+	for _, keyPath := range keyPaths {
+		cAdded := false
+		keyName := strings.TrimSuffix(filepath.Base(keyPath), constants.KeySuffix)
+		for net, id := range supportedNetworks {
```

Comment:
When would the user ever be able to use this address on the local network. Does adding that just add extra confusion?

Reply:
I maintain it could be argued that it's also weird to just print `fuji` when one is working locally only. One could in fact well be sending stuff via local RPC APIs, I have done that myself for testing.